astrosat/decorators.py

```diff
@@ -41,3 +41,34 @@ def _wrapper(request, *args, **kwargs):
         return _wrapper
 
     return _decorator
+
+
+def swagger_fake(fake_retval=None):
+    """
+    A decorator for a view that returns the provided value
+    if being run in the context of swagger schema generation;
+    (as per https://github.com/axnsan12/drf-yasg/issues/333#issuecomment-474883875)
+    this is intended to be applied to "get_queryset" and/or "get_object"
+    """
+    def _decorator(view_fn):
+
```

Comment:
I'm going to be contentious here and ask you not to add an empty line after function declarations... It means that code editors cannot collapse code. Really in the grand scheme of things, it likely doesn't matter, but in more complicated (longer files) it can be quite useful.

If you disagree, please say so and maybe why you think it is useful.

Reply:
Fine w/ me.

Reply:
Maybe there's some `yapf` setting that can enforce this?

Reply:
will look into it